Dockerfile.assisted-service-build

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.14.3
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.14
```

Comment:
Why not use `quay.io/app-sre/golang:1.14` here?
I think that's a straight mirror of the one we were using previously.